### ISSUE TITLE
Link to urgent care benefit webpage

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#5495]
## User Story:
- As a Veteran, I need to understand my urgent care benefits so that I can receive care without out of pocket cost.

## Objectives or Key Results this is meant to further
- Improve the Veteran Search Experience (Veterans have the information they need to be prepared for an urgent care visit. )
- Community Care Implementation

## Tasks
[ ] Add Alert to display between search boxes and map when 
1. Facility type = "Urgent care" and Service type drop down= "Community urgent care providers (in VA’s network)"
2. Facility type= "Community pharmacies (in VA’s network)",  

[ ]  Add alert text (similar to style in image below, **but with the following text**:

> **Important information about your Community Care appointment**
> Click below to learn how to prepare for your urgent care appointment with a Community Care provider.
> {button} Learn about VA urgent care benefit

[ ] Add button to link to webpage
[ ] Add placeholder link: https://www.va.gov/COMMUNITYCARE/programs/veterans/UrgentCareInstructions.asp

[ ] Get facility IDs to call for link

![image](https://user-images.githubusercontent.com/55411834/74374169-becea880-4d92-11ea-8b81-3f68a1a6df46.png)
****

## Acceptance Criteria:
[ ] Veterans using assistive technology can access link to eligibility info. 
[ ]When search parameters for **Facility type** = "Urgent care" and Service type drop down= "VA urgent care", alert **is not** displayed. 
[ ] When search parameters for **Facility type** = "Urgent care" and Service type drop down= "Community urgent care providers (in VA’s network)", alert and link to eligibility web page **are** displayed. 
[ ] When search parameters for **Facility type** = "Community pharmacies (in VA’s network)", alert and link to eligibility web page *are** displayed. 
[ ] Alert displays between search boxes and map and with the following language: 

> **Important information about your Community Care appointment**
> Click below to learn how to prepare for your urgent care appointment with a Community Care provider.
> {button} Learn about VA urgent care benefit

[ ] Alert displays with button which links to new webpage

**Comment by peggygannon** [comment]
Hello, @mmiddaugh and @arickalewis1 here is the content recommendation for this alert text. I have a few questions to make sure we have the messaging correct: 

- Will the new urgent care benefit webpage be created from the earlier PDF? If so, that PDF doesn't really list "eligibility requirements" for urgent care (it asks Veterans to call to find out if they're eligible). The PDF mainly lists what the Veteran will need to bring to their appointment. Will be new page actually list eligibility requirements? 

- Will we providing a link or a button for Veterans to "click" to go to this new page?  If it will be a button, then I'll need to rewrite the text below, as I've written that Veterans will be clicking a link. 
 
**Important information about your Community Care appointment**
Click here to learn more about how to prepare for your urgent care appointment with a Community Care provider.

**Comment by davidconlon** [comment]
@peggygannon to answer the question:

TL/DR - We aren't controlling the destination web content and may need to revise this text in a future iteration. It will be based upon the PDF but hopefully better, but we have no control.

Full context:
Office of Community Care is writing the page to be housed in their current teamsite.  
It will likely be based upon the PDF but, once the higher-ups saw the PDF, they were overwhelmed. 
So OCC is in the process of trying to clean it up _and_ make it a web page.  I'm not sure if we're actually going to be able to see the destination content before they start a fire-drill to make it live nor will we have a lot of influence over it I fear.

Now, that being said, It's likely on the roadmap to migrate to va.gov in the next 18 mos or so, so, here's hoping!(?)

**Comment by peggygannon** [comment]
ok, thank you @davidconlon for the context. 

I've written the alert based on what information appears on the PDF now.. (PDF does not include eligibility requirements). We can revisit this if we learn later that eligibility information will be on this new webpage.

**Comment by mmiddaugh** [comment]
@peggygannon I think they've decided to include a button and I notice your comment that the text recommended assumes a link. Can you update your text, please?

**Comment by mmiddaugh** [comment]
Sent Slack question to Mikki to find out requirements for url approval/review

**Comment by mmiddaugh** [comment]
> If the content page is not being created by a VFS team or hosted o n va.gov (new tech stack), then nope, no review from me. 
https://dsva.slack.com/archives/C0FQSS30V/p1581460805262000?thread_ts=1581460502.261700&cid=C0FQSS30V

**Comment by peggygannon** [comment]
@mmiddaugh @davidconlon @arickalewis1 

Please see updated content with button language. Let me know if you have any questions! 

**Important information about your Community Care appointment**
Click below to learn how to prepare for your urgent care appointment with a Community Care provider.
{button} Learn about VA urgent care benefit

**Comment by mmiddaugh** [comment]
@brangi 
We will receive the official link from the OCC in the next week or so and can anticipate slight language revisions. 
We'd like to get this work done now and moved to staging for testing. We will create a separate issue to address these anticipated changes before going live.

**Comment by mmiddaugh** [comment]
@brangi the OCC has confirmed this language. We can move forward with the PR process, just hold the deployment until we've confirmed the web page is ready.

**Comment by mmiddaugh** [comment]
Confirming the url:  https://www.va.gov/COMMUNITYCARE/programs/veterans/UrgentCareInstructions.asp

I realized there had been a typo in the Description previously.

**Comment by mmiddaugh** [comment]
Need to adjust language at top of the page when link is removed

> Need same-day care for a minor illness or injury? Search below to find your nearest VA health facility, VA-approved urgent care locations or pharmacies near you.

Moving this from separate [#6011](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6011)

**May need to re-estimate**

**Comment by mmiddaugh** [comment]
The url documented in this issue has been updated: [#6735](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6735)
